backend/app/controllers/api/v1/accounts/projects_controller.rb

```diff
@@ -57,15 +57,13 @@ def update_params
             :progress_impact_tracking,
             :description,
             :relevant_links,
-            :geometry,
```

Comment:
I thought I would just allow custom JSON, without having explicitly call `permit!`.

Reply:
How? :) I have not figured out way how to allow basically anything for `geometry` param including nested attributes...

Reply:
Ahh, sry, I though you had some idea how to do it different, because tbh, I don't like that `permit!` part at all, but all other options had problems with nested stuff 🤔

Reply:
No I don't, I thought it just should work as it was with jsonb column :D hehe